### PR TITLE
AG-1222 add SRM results to gene details

### PR DESCRIPTION
**Pull request by sagely1**

### CODE REVIEW

File: src/app/features/genes/components/gene-evidence-proteomics/gene-evidence-proteomics.component.ts [hunk 1]
```diff
@@ -74,10 +78,77 @@ export class GeneEvidenceProteomicsComponent {
...
+
+          SRMData.push({
+            key: data.tissue,
+            value: [data.min, data.median, data.max],
+            circle: {
+              value: item.log2_fc,
+              tooltip:
+                (item.hgnc_symbol || item.ensembl_gene_id) +
+                ' is ' +
+                (item.cor_pval <= 0.05 ? ' ' : 'not ') +
```
Comment (hallieswan) [inline src/app/features/genes/components/gene-evidence-proteomics/gene-evidence-proteomics.component.ts]: If the p-value is less than 0.05, should an empty string be used so there isn't an extra space in the sentence?

File: src/app/features/genes/components/gene-evidence-proteomics/gene-evidence-proteomics.component.ts [hunk 2]
```diff
@@ -74,10 +78,77 @@ export class GeneEvidenceProteomicsComponent {
       this.selectedUniProtId = this.uniProtIds[0];
     }
 
+    this.initSRM();
+
     this.initLFQ();
     this.initTMT();
   }
 
+  initSRM() {
```
Comment (hallieswan) [inline src/app/features/genes/components/gene-evidence-proteomics/gene-evidence-proteomics.component.ts:113]: `initSRM` looks similar to `initLFQ` and `initTMT` -- would it be possible to extract the shared logic into a helper method?

**Review by hallieswan — APPROVED** [review]
Looks good to me! Just a couple small questions

File: src/app/features/genes/components/gene-evidence-proteomics/gene-evidence-proteomics.component.ts (hunk 1 above)
Comment (sagely1) [reply src/app/features/genes/components/gene-evidence-proteomics/gene-evidence-proteomics.component.ts]: Nice find.  I will remove it and refactor this as it is used in multiiple places.

File: src/app/features/genes/components/gene-evidence-proteomics/gene-evidence-proteomics.component.ts (hunk 2 above)
Comment (sagely1) [reply src/app/features/genes/components/gene-evidence-proteomics/gene-evidence-proteomics.component.ts:113]: Yes, I'll do the refactor as all three seem to share the same logic.

**Pull request merged**